### PR TITLE
Set tilemask automatically

### PR DESCRIPTION
**Pull request by Variable-ind**
This will automatically set a tilemask whenever we change project or cell,
No need for setting it manually
This also removed the need for it to be stored in a Project

### CODE REVIEW

File: src/Main.tscn [hunk 1]
```diff
@@ -64,6 +64,7 @@ __meta__ = {
 [node name="CreateNewImage" parent="Dialogs" instance=ExtResource( 28 )]
 
 [node name="OpenSprite" parent="Dialogs" instance=ExtResource( 12 )]
+rect_min_size = Vector2( 170, 59.5 )
```
Comment (OverloadedOrama) [inline src/Main.tscn:67]: I'm guessing this is just Godot adding unnecessary changes to Main.tscn again? :P

Comment (Variable-ind) [reply src/Main.tscn:67]: Hmmm, i think so (i thought i prevented main from getting committed 😅) feel free tho remove them 😅

**Review by OverloadedOrama — APPROVED** [review]
I think this is a nice change. Thank you!
